keps/sig-node/00014-resource-api.md

```diff
@@ -0,0 +1,368 @@
+---
+kep-number: 14
+title: New Resource API Proposal
+authors:
+  - "@vikaschoudhary16"
+  - "@jiayingz"
+owning-sig: sig-node
```

Comment:
I'd add sig-scheduling as well since there is quite some impact to scheduling, quota, etc.

Reply:
done